### PR TITLE
Hello Microverse project

### PR DESCRIPTION
**Pull request by Tchilo**
Add index.html file with one heading tag and text.
Add styles.css file with one rule.
Add test.md file and add to .gitignore file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+
+-CSS
+- HTML
 
 ## Live Demo
```
Comment (safafa) [inline README.md]: - [ ] [OPTIONAL] It's good to use a Live demo so you can show people your work, always make sure on having a working link.

**Review by safafa — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED ♻️ 

Hi @Tchilo , You did a great job so far 👏🏼 👏🏼 

- Commit messages are meaningful and professional.
- You set up the linters correctly and all the checks are passing.
- the Pull request comment is descriptive.

However, here are some points that you need to work on to get your project approved. You're almost there Keep up the good work !

## Required changes :

Check the inline comments.

- [x] Kindly modify the pull request title as it should also be related to the current changes.

## Optional changes :
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by Tchilo** [comment]
hello, thank you very much for your code review am a little behind on most
of the things you have told me to change and they are not specified in the
requirements for this project. I will change what I can. I hope you can
help me where I won't be able to change

File: README.md (hunk 1 above)
Comment (Tchilo) [reply README.md]: I have done everything except this one

**Review by Emmyn5600 — APPROVED** [review]
## STATUS: APPROVED :muscle::clap:

Hi, @Tchilo 

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous code reviewer.

- To highlight:
   * #### You have the descriptive Readme:heavy_check_mark:
   * #### Your commits look more professional:heavy_check_mark:
   * #### Linters are passing successfully.:heavy_check_mark:
   
- Optional suggestions:

* #### Use the descriptive name for the feature branch
* #### Use past tense while you are creating the PR, for example Added index.html file instead of Add index.html file.
   
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**